src/iterative_sorting/iterative_sorting.py

```diff
@@ -5,22 +5,33 @@ def selection_sort( arr ):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
+        # (hint, can do in 3 loc)
 
+        #iterating through array, adding one to the index (i) each time 
+        for j in range(i+1, len(arr)):
+            #comparing current index (j) to smallest_index to see if j is smaller
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
         # TO-DO: swap
-
-
-
+        #swapping minimum element with the first element
+        arr[i], arr[smallest_index] = arr[smallest_index], arr[i]
 
     return arr
 
 
-# TO-DO:  implement the Bubble Sort function below
+# TO-DO:  implement the Bubble Sort function below!
 def bubble_sort( arr ):
 
+    #Here, we are iterating over each element in the arr, using 'i' to represent each individual element
```

Comment:
Be sure you have clarity when you're accessing an element in a list like arr[i] vs  dealing with the index i